docs/UserGuide.md

```diff
@@ -2,31 +2,46 @@
 layout: page
 title: User Guide
 ---
+This guide serves to provide a reference for first-time users to get familiar with the `ZooKeep` app.
 
-1. Table of Contents
+* Table of Contents
 {:toc}
 
 ## Introduction
-ZooKeep is a desktop app for managing animals under a zookeeper’s care, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, ZooKeep can get your management tasks done faster than traditional GUI apps.
+`ZooKeep` is a desktop app for managing animals under a zookeeper’s care, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, `ZooKeep` can get your management tasks done faster than traditional GUI apps.
+
+## Legend
+
+<div markdown="block" class="alert alert-info">
```

Comment:
sorry i think the reason why the spacing is weird is bc it is a block. could you change it to try changing it to `markdown="span"`?